### PR TITLE
Add govuk_sidekiq to organisation publisher

### PR DESCRIPTION
**Pull request by MahmudH**
The gem was already added. What remains is to add a config file and Procfile.

https://trello.com/c/fn3tQUmR/150-add-sidekiq-worker-to-org-publisher

### CODE REVIEW

**Review by rubenarakelyan — APPROVED** [review]

File: config/Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+worker: bundle exec sidekiq -C ./config/sidekiq.yml
```
Comment (cbaines) [inline config/Procfile]: You're missing the ending newline in this file.

File: config/sidekiq.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+:concurrency: 2
```
Comment (cbaines) [inline config/sidekiq.yml]: And this one too.

File: config/Procfile (hunk 1 above)
Comment (Davidslv) [reply config/Procfile]: @MahmudH see if this helps https://stackoverflow.com/questions/12415147/ensure-newline-at-eof-in-rubymine

Comment (MahmudH) [reply config/Procfile]: 👍

File: config/sidekiq.yml (hunk 2 above)
Comment (MahmudH) [reply config/sidekiq.yml]: 👍

**Review by Davidslv — APPROVED** [review]
👍 merge when the tests are green

**Pull request merged**